sale_order_display_country/models/sale_order.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    country = fields.Many2one(
+        related="partner_id.country_id", readonly=True, store=True,
+    )
```

Comment:
```suggestion
    country_id = fields.Many2one(related="partner_id.country_id", store=True)
```
Should be readonly=True by default for compute/related fields.